client.go

```diff
@@ -62,11 +63,24 @@ func (c *Client) QueryRaw(req []byte) (result []byte, err error) {
 		clientTimeout = c.Timeout
 	}
 
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+	}
+
 	httpClient := &http.Client{
-		Timeout: clientTimeout,
+		Timeout:   clientTimeout,
+		Transport: tr,
+	}
+
+	request, err := http.NewRequest("POST", "https://broker.proto.npav.accedian.net/druid/v2?pretty=", bytes.NewBuffer(req))
```

Comment:
Hardcoded URL? :)

Can we make it a config, and have it default to "https://broker" for druid broker?

Also, you should go to `https://broker/druid/v2?pretty`

We don't want this request to hit our external load balancer

Reply:
oh god...yeah I forgot to change this..